<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
font: 14px Poppins;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="inputPlace">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>321</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 1px;
background-color: rgb(234, 234, 234);</string>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
      <property name="placeholderText">
       <string>Ej: Colegio de Manaure</string>
      </property>
     </widget>
     <widget class="QWidget" name="widgetKeyboard" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>480</width>
        <height>135</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(234, 234, 234);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>8</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nombre de la muestra&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextBtn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>61</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #22577a;
color: white;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="itRainedBtn">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
border: 2px solid grey;
border-radius: 6px;</string>
      </property>
      <property name="text">
       <string>¿Llovió recientemente?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>61</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #22577a;
color: white;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="gpsBtn">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>180</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: #00007f;
color: white;
border: none;
border-radius: 15px;

</string>
      </property>
      <property name="text">
       <string>Localizar con GPS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Latitud, longitud&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="selectFolderBtn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
color: #00007f;
border: 2px solid #00007f;
border-radius: 15px;

</string>
      </property>
      <property name="text">
       <string>Seleccionar carpeta</string>
      </property>
     </widget>
     <widget class="QLabel" name="folderLbl">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Carpeta:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Origen o fuente de la muestra</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setManualLocationBtn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>180</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
color: #00007f;
border: 2px solid #00007f;
border-radius: 15px;

</string>
      </property>
      <property name="text">
       <string>Ingresar manualmente</string>
      </property>
     </widget>
     <widget class="QPushButton" name="originBtn">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>30</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
border: 1px solid black;
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>Escoja una opción</string>
      </property>
     </widget>
     <widget class="QPushButton" name="prevBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>61</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #22577a;
color: white;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="saveBtn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>61</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #22577a;
color: white;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="openCreateFolderBtn">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>230</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: #00007f;
color: white;
border: none;
border-radius: 15px;
</string>
      </property>
      <property name="text">
       <string>Crear una nueva carpeta</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>431</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: opx;</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>431</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QSlider" name="verticalSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>31</width>
        <height>181</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:vertical{ 
    background-color: rgb(234, 234, 234);
    width: 30px;
    border-radius: 10px;
}

QSlider::handle:vertical { 
    background-color: #00007f;
   	width: 20px;
	height: 50px;
	margin-top: 0px; 
	margin-bottom: 0px;
    border-radius: 10px;
}

QSlider{
    background-color: rgb(234, 234, 234);
    border-radius: 10px;
}
</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="pageStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
      <property name="invertedControls">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="emptyFoldersNoticeLbl">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: grey;
font-size: 24px;</string>
      </property>
      <property name="text">
       <string>No hay carpetas guardadas</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>38</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nombre&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="folderName">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>241</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 1px;
border-radius: 6px;
font: 14px &quot;Poppins&quot;;
background-color: rgb(234, 234, 234);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
      <property name="placeholderText">
       <string>Ej: Muestras Colegio La Rosa</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Nueva carpeta&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="widgetKeyboard2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>480</width>
        <height>135</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(234, 234, 234);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createFolderBtn">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: #00007f;
color: white;
border: none;
border-radius: 15px;
</string>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
